projects/angular/src/forms/file-input/file-input-validator.ts

```diff
@@ -29,7 +29,7 @@ export class ClrFileInputValidator implements Validator {
       errors.required = true;
     }
 
-    const accept = fileInputElement.accept?.split(',').map(type => type.trim());
+    const accept = fileInputElement.accept ? fileInputElement.accept.split(',').map(type => type.trim()) : null;
```

Comment:
Instead of `null` set `accept` to empty array `[]`. Then you will not need the null check.
 `accept?.length` -> `accept.length`

```suggestion
    const accept = fileInputElement.accept ? fileInputElement.accept.split(',').map(type => type.trim()) : [];
```

Reply:
I think `null` is more appropriate here. It doesn't accept zero file types. Instead, there is no accept validation at all. I'll remove the `length` check.

Reply:
This is updated.